src/Drivers/Jibit/JibitFileCache.php

```diff
@@ -0,0 +1,500 @@
+<?php
+namespace Shetabit\Multipay\Drivers\Jibit;
+
+/**
+ * PhpFileCache - Light, simple and standalone PHP in-file caching class
+ * This class was heavily inspired by Simple-PHP-Cache. Huge thanks to Christian Metz
+ * @license MIT
+ * @author Wruczek https://github.com/Wruczek
+ */
+class JibitFileCache
```

Comment:
copying a package file into the project is not a good practice! during the time the package should be maintained and by releasing new version of PHP it should be updated and needs some fixes.

btw, please remove this file and use https://github.com/chillerlan/php-cache instead. please install it using composer in the project.

Reply:
I integrated the Jibit driver into this package based on their sample code recommendation from:
https://newdashboard.jibit.ir/assets/documents/Jibit-class-v3.zip

However, upon further consideration, I realize it may not align with best practices. I'm planning to refactor and integrate the php-cache repository into this driver for better long-term maintainability.

Reply:
It's addressed in the next commits.